docs/readthedocs/models/input/grid/transformer2w.md

```diff
@@ -10,41 +10,129 @@ It is assumed, that node A is the node with higher voltage.
-|:----------------|:-----|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| uuid            | --   |                                                                                                                                                                               |
-| id              | --   | Human readable identifier                                                                                                                                                     |
-| operator        | --   |                                                                                                                                                                               |
-| operationTime   | --   | Timely restriction of operation                                                                                                                                               |
-| nodeA           | --   | Higher voltage node                                                                                                                                                           |
-| nodeB           | --   | Lower voltage node                                                                                                                                                            |
-| parallelDevices | --   | overall amount of parallel transformers to automatically <br> construct (e.g. parallelDevices = 2 will build a <br> total of two transformers using the specified parameters) |
-| type            | --   |                                                                                                                                                                               |
-| tapPos          | --   | Current position of the tap changer                                                                                                                                           |
-| autoTap         | --   | true, if there is a tap regulation apparent and active                                                                                                                        |
+```{eval-rst}
+.. list-table::
+   :widths: 33 33 33
+   :header-rows: 0
+
+   * - Attribute
+     - U n i t
+     - Remarks
+
+   * - uuid
+     - –
+     -
+
+   * - id
+     - –
+     - Human readable identifier
+
+   * - operator
+     - –
+     -
+
+   * - operationTime
+     - –
+     - Timely restriction of operation
+
+   * - nodeA
+     - –
+     - Higher voltage node
 
+   * - nodeB
+     - –
+     - Lower voltage node
+
+   * - parallelDevices
+     - –
+     - overall amount of parallel transformers to automatically
+         construct (e.g. parallelDevices = 2 will build a total of
+         two transformers using the specified parameters)
```

Comment:
This is not rendered correctly in the docs.

Reply:
Check